contributors/devel/api_changes.md

```diff
@@ -397,22 +404,22 @@ used.
 ### Edit types.go
 
 Similar to the versioned APIs, the definitions for the internal structs are in
-`pkg/api/types.go`. Edit those files to reflect the change you want to make.
-Keep in mind that the internal structs must be able to express *all* of the
-versioned APIs.
+`pkg/apis/<group>/types.go`. Edit those files to reflect the change you want to
+make.  Keep in mind that the internal structs must be able to express *all* of
+the versioned APIs.
 
 ## Edit validation.go
 
 Most changes made to the internal structs need some form of input validation.
 Validation is currently done on internal objects in
-`pkg/api/validation/validation.go`. This validation is the one of the first
-opportunities we have to make a great user experience - good error messages and
-thorough validation help ensure that users are giving you what you expect and,
-when they don't, that they know why and how to fix it. Think hard about the
-contents of `string` fields, the bounds of `int` fields and the
+`pkg/apis/<group>/validation/validation.go`. This validation is the one of the
+first opportunities we have to make a great user experience - good error
+messages and thorough validation help ensure that users are giving you what you
+expect and, when they don't, that they know why and how to fix it. Think hard
+about the contents of `string` fields, the bounds of `int` fields and the
 requiredness/optionalness of fields.
```

Comment:
Could requiredness/optionalness be replaced by "optionality" (a dictionary term)?